Sources/OktaLogger/Analytics/Analytics.swift

```diff
@@ -0,0 +1,214 @@
+//
+//  AnalyticsManager.swift
+//  OktaLogger
+//
+//  Created by SP on 8/9/22.
+//  Copyright © 2022 Okta, Inc. All rights reserved.
+//
+
+import Foundation
+import AppCenterAnalytics
+
+@objc public enum Provider: Int {
```

Comment:
Can we drop @objc conformance? Do we really need it?

Reply:
Not sure, probably OV has objective-C code and we might need to track the events on that class